srcpkgs/luakit/template

```diff
@@ -1,8 +1,8 @@
 # Template file for 'luakit'
 pkgname=luakit
 reverts="2017.08.10_1"
-version=2.3.6
-revision=2
+version=2.4.0
+revision=1
 conf_files="/etc/xdg/luakit/*.lua"
```

Comment:
I don't think this works. (you need to specify the full paths manually for xbps to avoid overwriting the config files)